Fix(Insomnia): update with troubleshooting steps for timeouts only in Insomnia (#4869)
Kong/developer.konghq.com: app/insomnia/requests.md

- [Inso CLI overview](/inso-cli/)
- [Inso CLI configuration](/inso-cli/configuration/)

### Troubleshooting

> If you encounter request timeouts errors (400) only in Insomnia, the `Content-Lenght` parameter might be forcing the server to validate the content length against the body, and failing in case it mutates.

Reviewer: @juliamrch Is this copilot suggestion worth looking into?

> If you encounter request timeouts errors (400) only in Insomnia, the `Content-Lenght` parameter might be forcing the server to validate the content length against the body, and failing in case it mutates.

Reviewer: ```suggestion
If you encounter request timeouts errors (400) only in Insomnia, the `Content-Length` parameter might be forcing the server to validate the content length against the body, and failing in case it mutates.
```

> If you encounter request timeouts errors (400) only in Insomnia, the `Content-Lenght` parameter might be forcing the server to validate the content length against the body, and failing in case it mutates.

Reviewer: ```suggestion
If you get a `400` request timeouts error in Insomnia, but the same request works with cURL, it may be due to the `Content-Length` header. Insomnia might be sending a fixed `Content-Length`, causing the server to validate it against the body, and failing if they don't match.
```

> If you encounter request timeouts errors (400) only in Insomnia, the `Content-Lenght` parameter might be forcing the server to validate the content length against the body, and failing in case it mutates.

Author: Yes, it's more accurate.

To resolve the error, try the following:

> 1. From your request's **Mock Header** tab, remove or deactivate the `Content-Lenght` parameter.

Reviewer: ```suggestion
1. From your request's **Mock Header** tab, remove or deactivate the `Content-Length` parameter.
```
